cli/src/pcluster/config/cluster_config.py

```diff
@@ -1183,6 +1195,102 @@ def __init__(
         self.on_node_updated = Resource.init_param(on_node_updated)
 
 
+class LoginNodeImage(Resource):
+    """Represent the configuration of LoginNode Image."""
+
+    def __init__(self, custom_ami: str, **kwargs):
+        super().__init__()
+        self.custom_ami = Resource.init_param(custom_ami)
+
+    def _register_validators(self, context: ValidatorContext = None):  # noqa: D102 #pylint: disable=unused-argument
+        if self.custom_ami:
+            self._register_validator(CustomAmiTagValidator, custom_ami=self.custom_ami)
+
+
+class LoginNodeSsh(Ssh):
+    """Represent the SSH configuration for LoginNodes."""
+
+    def __init__(
+            self,
+            key_name: str,
+    ):
+        super().__init__(key_name=key_name)
+
+    def _register_validators(self, context: ValidatorContext = None):  # noqa: D102 #pylint: disable=unused-argument
+        super()._register_validators(context)
+
+
+class LoginNodeNetworking(_BaseNetworking):
+    """Represent the networking configuration for the login node."""
+
+    def __init__(
+            self,
+            subnet_id: str,
+            security_groups: List[str] = None,
+            additional_security_groups: List[str] = None,
+            **kwargs,
+    ):
+        super().__init__(security_groups=security_groups, additional_security_groups=additional_security_groups,
+                         **kwargs)
+        self.subnet_id = Resource.init_param(subnet_id)
+
+    def _register_validators(self, context: ValidatorContext = None):
+        super()._register_validators(context)
+        self._register_validator(AvailabilityZoneValidator, login_node_subnet_id=self.subnet_id,
+                                 head_node_subnet_id=HeadNodeNetworking.getInstance().subnet_id)
+
+    @property
+    def availability_zone(self):
```

Comment:
Where is this property used? If it's unused then we can remove it.

Reply:
It was originally intended for availability_zone validator, but I modified the validator implementation so it's not needed now, and I forgot to remove it. Now removed.